index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
+  <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  
+  <main class="main">
+ 
+<header class="nav-bar">    
+  <div class="logo-left">My Logo</div>
+ <i class="material-icons">menu</i>
+</header>
+
+<section class="headline">
+   <h1>I'm Cookey<br>Glad to see <br> you!</h1>
```

Comment:
- [ ] Your h1 tag is not matching the Figma design as you haven't used the font family and color as Figma design so kindly implement it.
### your design
![hisssh1](https://user-images.githubusercontent.com/81727988/170711209-1a7cbda4-e762-4f1b-95c2-06d6d6440e02.png)
### Figma design
![figma h1](https://user-images.githubusercontent.com/81727988/170711252-84c53d86-bb9c-41bf-bec8-44a740829252.png)